src/envoy/http/service_control/config_parser.h

```diff
@@ -98,7 +112,8 @@ class FilterConfigParser {
       const {
     return config_;
   }
-  const RequirementContext* FindRequirement(absl::string_view operation) const {
+  const RequirementContext* find_requirement(
```

Comment:
I want c++ code under "envoy" to follow envoy style. which is findRequirement

Reply:
Let me do this in another PR across the entire directory. 

This change makes it consistent with the rest of this file at least.

Reply:
OK